src/hotspot/share/gc/shenandoah/shenandoahEvacOOMHandler.inline.hpp

```diff
@@ -26,6 +26,7 @@
 #define SHARE_GC_SHENANDOAH_SHENANDOAHEVACOOMHANDLER_INLINE_HPP
 
 #include "gc/shenandoah/shenandoahEvacOOMHandler.hpp"
+
```

Comment:
Why the blank line?

Reply:
Just eliminates 1-line diff from upstream.